### ISSUE TITLE
Bug/PostgreSLQ: Compatiblity Issue with PostgresSQL 13

### ISSUE THREAD

**Issue by bennet-esyoil** [crowdsecurity/crowdsec#1182]
**Describe the bug**
Running CrowdSec with PostgreSQL 13 breaks `cscli decision list -a`

**To Reproduce**
1. Install CrowdSec with PostgreSQL 13
2. run `cscli decisions list -a`

**Expected behavior**
Get list of decisions

**Technical Information (please complete the following information):**
 - OS: Ubuntu 21.10
 - Version **guess: latest 1.2.3**

**Additional context**
```
FATA[18-01-2022 11:02:59 PM] Unable to list decisions : performing request: API error: unable to count nb alerts: pq: column "alerts.created_at" must   appear in the GROUP BY clause or be used in an aggregate function
```

**Comment by AlteredCoder** [comment]
Hello @bennet-esyoil, thanks for your issue.

It has been fixed in https://github.com/crowdsecurity/crowdsec/pull/1184 and will be fixed in the next release :)

**Comment by bennet-esyoil** [comment]
@AlteredCoder I think you can write "closes #1182" in the PR-Description to auto-close the issue when the PR is merged, for the next time!
